machinery/ast.js

```
@@ -1,53 +1,38 @@
-  isRootJSXElement, hasParentsJSXElementsWithClassName, isInJSXBranch, isInExport,
+  isInExport
 }
 
 function getPropertyName(property) {
+  if (!property) return '???'
   switch (property.type) {
     case 'Identifier': return property.name
     case 'Literal': return property.value
     case 'BinaryExpression': return getPropertyName(property.left)
     case 'TemplateLiteral':
       const [name] = property.quasis
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)